docs/game/cheat_sheet.md

```diff
@@ -12,37 +12,42 @@ enjoyable, entertaining and informative show. Winning isn't everything.
 Manners
 -------
 
-Some general manners guidelines for panel games, and specifically Just A Minute
+Panel games, much like the greatness of Great Britain, are founded on
+good manners. Here are some guidelines that make Just A Minute flow smoothly.
```

Comment:
We may need to tone-down the patriotism, many of the intended consumers are from other nations.